explorer/vite.config.js

```diff
@@ -0,0 +1,57 @@
+// eslint-disable-next-line import/no-unresolved
+import { sveltekit } from "@sveltejs/kit/vite";
+import { defineConfig, loadEnv } from "vite";
+import basicSsl from "@vitejs/plugin-basic-ssl";
```

Comment:
Maybe you don't want the hassle of `basicSsl` if you don't use the Crypto API, or need HTTPS for other reasons.